plotly/Cargo.toml

```
@@ -20,7 +20,7 @@ plotly_image = ["image"]
 wasm = ["getrandom", "js-sys", "wasm-bindgen", "wasm-bindgen-futures"]
 
 [dependencies]
-askama = { version = "0.11.0", features = ["serde-json"] }
+askama = { version = ">=0.11.0, <=0.12.0", features = ["serde-json"] }
```

I don't think we'd want to prohibit a `0.12.1` release.
```suggestion
askama = { version = ">=0.11.0, <0.13.0", features = ["serde-json"] }
```